modules/user_passwords/module.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "User Password Verifier",
```

Comment:
add missing metadata to this module. add a tag implying this module is internal + specify in description how it' used.